pkg/ingester/tailer.go

```diff
@@ -61,15 +59,16 @@ func newTailer(orgID, query string, conn TailServer) (*tailer, error) {
 	matchers := expr.Matchers()
 
 	return &tailer{
-		orgID:          orgID,
-		matchers:       matchers,
-		pipeline:       pipeline,
-		sendChan:       make(chan *logproto.Stream, bufferSizeForTailResponse),
-		conn:           conn,
-		droppedStreams: []*logproto.DroppedStream{},
-		id:             generateUniqueID(orgID, query),
-		closeChan:      make(chan struct{}),
-		expr:           expr,
+		orgID:             orgID,
+		matchers:          matchers,
+		pipeline:          pipeline,
+		sendChan:          make(chan *logproto.Stream, bufferSizeForTailResponse),
+		conn:              conn,
+		droppedStreams:    []*logproto.DroppedStream{},
```

Comment:
Since we have `maxDroppedStreams`, we should make this an array equal to that length

Reply:
sure indeed. Done!